tomls/omnibus-base-mainnet-andromeda/perps/global.toml

```diff
@@ -5,19 +5,19 @@ defaultValue = "1"
 defaultValue = "30000"
 
 [setting.perps_keeper_l2_cost_settle_gas_units]
-defaultValue = "1200000"
+defaultValue = "1300000"
```

Comment:
I'm stuck in a forever-loop

![image](https://github.com/Synthetixio/synthetix-deployments/assets/28145325/b1b26f50-28f9-434e-8eb3-a4508f28304e)

![image](https://github.com/Synthetixio/synthetix-deployments/assets/28145325/7a7fd2d3-b599-418a-a57d-839f344cf8dd)

Reply:
![image](https://github.com/Synthetixio/synthetix-deployments/assets/62237347/7a203e4b-388c-4489-ad7c-916557a98e34)

Reply:
![IMG_2532](https://github.com/Synthetixio/synthetix-deployments/assets/28145325/24cd4302-669f-471d-9d57-1eaf912779be)